dev/CommonStyles/Common_themeresources.xaml

```diff
@@ -38,7 +38,7 @@
             <x:Boolean x:Key="UseSystemFocusVisuals">True</x:Boolean>
             <Thickness x:Key="TextControlBorderThemeThickness">1</Thickness>
             <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
-            <Thickness x:Key="TextControlThemePadding">10,6,6,5</Thickness>
+            <Thickness x:Key="TextControlThemePadding">10,5,6,6</Thickness>
```

Comment:
Should these non-`SolidColorBrush` resources be moved outside of individual themes to to avoid duplication?

Reply:
Honestly, I'm not sure 100% sure why it's here, but I assume there was a reason. There are some cases where thicknesses differ between light/dark themes and high contrast. If I moved any of it now it could mess up things if 3rd parties were overriding these values in theme dictionaries.

Reply:
Reading other PRs, I know a lot of other resources have been moved outside of the theme dictionaries recently. My thinking is that if they're gonna break those specific cases already anyways, we might as well clean up the templates across the repo? 😂

Reply:
These ones have the word "theme" in it, though. As I can't really change the name either, I think they should stay in the theme dictionaries for now.